Feature/api

juandave25 opened the pull request:
branch with api

juandave25 committed:
backend development integrado con front end

juandave25 committed:
Merge branch 'master' into feature/API

juandave25 committed:
modifications

juandave25 committed:
endpints backend

juandave25 committed:
modifications  pipo back and front

juandave25 committed:
make it better

juandave25 committed:
deleting backend project

juandave25 committed:
Add endpoint login

juandave25 committed:
fix conflicts

juandave25 committed:
multilanguage

juandave25 committed:
fixing in language view

juandave25 committed:
Merge branch 'master' into feature/API

# Conflicts:
#	package-lock.json
#	src/components/question/questionModel.js

JoseVf commented on src/components/Languages/languageView.js:
```diff
@@ -0,0 +1,66 @@
+            {global.lang.map((item,key)=>(
+                <ListItem icon>
+                    <Left>
+                    <Button style={{ backgroundColor: "#007AFF" }} key={key}  onPress={() => this.setText(item.shortform)}>
```
El evento de `onPress` debería tenerlo el `ListItem`, de esa forma puedo seleccionar presionando en cualquier parte de la fila y no solamente en el icono.

JoseVf commented on src/components/login/loginModel.js:
```diff
@@ -22,10 +22,22 @@ class LoginModel {
+  .then(response => response.json())
+  .then(responseJson =>{
+    console.log(responseJson.profileData);
+    if(view.state.username!=responseJson.profileData.usuario){
```
Al dar click en iniciar sesión sale un error de `undefined is not an object (evaluating 'responseJson.profileData.usuario')`

JoseVf commented on src/components/Languages/languageView.js:
```diff
@@ -0,0 +1,66 @@
+    super(props);
+    const lang = [
+        { shortform: 'es', longform: 'Español' },
+        { shortform: 'en', longform: 'Inglés' },
```
Pienso que el `longform` debería contener el idioma en su idioma, por ejemplo en este caso ̣*English*

JoseVf commented on src/components/login/loginController.js:
```diff
@@ -84,6 +84,10 @@ class LoginController {
   }
 
+  play(){
+    view.replace('theme');
```
Esta función debería estar en el model, en el controller quedaría solamente `model.play()`

juandave25 committed:
refactoring and adding new functions

juandave25 committed:
Merge branch 'master' into feature/API

# Conflicts:
#	src/components/gameMode/gameModeModel.js

juandave25 committed:
Addng flags

juandave25 committed:
Fixes flags

juandave25 committed:
it was fixed the flags and the language window for show the game languages
read me file changes for the new installation in future repository clonings
changes in android manifest and files for new version of react native and the new implementations work correctly

juandave25 committed:
debugging
